responses/validation.go

```diff
@@ -45,6 +45,17 @@ func NewValidationErrorResponse(err error) *ValidationErrorResponse {
 	r.Code = constants.VAL_VALIDATOR_ERROR
 	r.Message = "Validation error"
 
+	if len(messages) > 0 {
+		r.Message = ""
+		for i, msg := range messages {
+			r.Message += msg
+
+			if i < len(messages)-1 {
+				r.Message += " "
+			}
+		}
+	}
+
```

Comment:
Może powinniśmy przenieść to do osobnego "konstruktora"? Ta funkcja tworzy response przeznaczony do walidacji na podstawie bindingu jsona. Moglibyśmy zrobić osobną funkcję, która przyjmuje listę pól i message i wstawia to do formatu validation error z API.

Reply:
dobra, cofam tę zmianę stąd